packages/docs/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
```

Comment:
We should remove the changes from "export" and "build/utils" from this PR as they are orthogonal changes.

Reply:
Also the preview deployment is broken on the current PR.